projects/plugins/boost/app/modules/Modules_Setup.php

```diff
@@ -61,6 +61,16 @@ public function get_status() {
 		return $status;
 	}
 
+	public function get_sub_modules_state() {
+		$state = array();
+		foreach ( $this->available_modules as $slug => $module ) {
+			if ( $module->has_sub_modules() ) {
+				$state[ $slug ] = $module->feature->get_sub_modules_state();
```

Comment:
I'm not sure I like that I'm asking if the module has sub modules, but then referring to the feature.